lib/IRGen/IRGenSIL.cpp

```diff
@@ -1938,59 +1972,129 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
 
   auto &silModule = CurSILFn->getModule();
 
+  auto *TFNewStatusFn = IGM.getTF_NewStatusFn();
+  auto status = Builder.CreateCall(TFNewStatusFn, {});
+
+  if (opInfo.getOperationName() != "Const") {
```

Comment:
Special casing the Const op is just a temporary thing for your incremental development, right?

Reply:
Yup! There was a TODO above.